### PR TITLE
WebXRManager: Clear state and added some accessor functions

### PR DESCRIPTION
**Pull request by cabanier**
Some of the state wasn't cleaned up in the WebXR manager which could lead to errors.
Also added some helper to give easier access to WebXR state

This contribution is funded by [Oculus](https://oculus.com).

### CODE REVIEW

File: src/renderers/webxr/WebXRManager.js [hunk 1]
```diff
@@ -188,6 +193,24 @@ class WebXRManager extends EventDispatcher {
...
+
+		};
+
+		this.getBinding = function () {
+
+			return glBinding;
+
+		};
+
+		this.getxrFrame = function () {
```
Comment (mrdoob) [inline src/renderers/webxr/WebXRManager.js:208]: Isn't `frame` available already in the `onAnimationFrame()` callback?

Comment (cabanier) [reply src/renderers/webxr/WebXRManager.js:208]: Yes, but then it has to be passed into the components that would like to have access.
For instance, I was experimenting with creating a cube layer for the background and it was handy to have it available on the `xr` object.

Comment (cabanier) [reply src/renderers/webxr/WebXRManager.js:208]: @mrdoob I can remove it if you prefer

Comment (mrdoob) [reply src/renderers/webxr/WebXRManager.js:208]: I see...

How following the naming pattern of the other methods and call it `getFrame()`?

Comment (cabanier) [reply src/renderers/webxr/WebXRManager.js]: done

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**